specification/compute/resource-manager/readme.md

```diff
@@ -0,0 +1,87 @@
+# Compute

```

Comment:
Is there a definitive reason one way or the other?

I have no strong feelings one way or the other.